src/scheduler/buckets.cpp

```diff
@@ -37,6 +37,7 @@
  * subject to Altair's trademark licensing policies.
  */
 
+#include <algorithm>
```

Comment:
just curious, what exactly is this needed for?

Reply:
Oh, it's for std::find().  I should remove it from a number of files.